lib/components/narrative/default/format-time.js

```diff
@@ -0,0 +1,14 @@
+import moment from 'moment-timezone'
```

Comment:
Since this component is also used in other kinds of itineraries that aren't the default, this should live somewhere else, perhaps even in some common components folder.

Reply:
see above